ingredient/src/index.ts

```diff
@@ -0,0 +1,25 @@
+import { json, urlencoded } from 'body-parser';
+import * as dotenv from 'dotenv';
+import * as express from 'express';
+import { db } from './db';
+
+dotenv.config();
+
+const app = express();
+
+app.use(json());
+app.use(urlencoded({ extended: true }));
+
+// test query
+app.get('/', (_, res) => {
+    db.query('SELECT * from test', (error, results) => {
+        if (error) {
+            res.status(500).send('Internal server error');
+        }
+        res.status(200).json(results.rows);
+    });
+});
+
+app.listen(3000, () => {
+    console.log('Server listening on port 3000');
+});
```

Comment:
3000 is set for gateway. Can you select a new one for the ingredient service? Please coordinate with @no-or  @andrewzulaybar , and @ginsstaahh . It won't really matter once they are in docker containers but having the option to run all these servers without them should work without messing with it more.